tip-wlan/charts/opensync-mqtt-broker/values.yaml

```diff
@@ -80,17 +80,13 @@ ingress:
   #    hosts:
   #      - chart-example.local
 
-resources: {}
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+resources:
+  limits:
+    cpu: 100m
+    memory: 10Mi
```

Comment:
@brennerm as discussed, in places where we need to go to such resolutions, lets be more conservative and keep to a minimum of 0.5 cpu and 128M